### ISSUE TITLE
Fluencerator Force-restart? 

### ISSUE THREAD

**Issue by DoktorPlekter** [NikolaiVChr/Autinn#19]
Hey! I've been using the Fluencerator alot in my patches, and I absolutely love the beautiful and organic phrases it creates. 

Correct me if i'm wrong about this, but there seem to be no way to force an **instant** trigger of the "Generate" button, so you always have to wait for the phrase to end before it can generate a new melody (f.eks in a new incoming scale). 
This also makes it hard to properly sync the Fluencerator with other Fluencerators and melody-generators, which might be a problem when dealing with different musical scales.. 

I'd suggest there be some type of "Force generate" input or switch, so a new phrase will generate **right away** without having to wait :)

**Comment by NikolaiVChr** [comment]
Yes there is yet no way to start a new phrase before the current has played out.

There is 3 principles in it, that has to do with how humans perceive note sequences. Those can potentially be broken by starting a new phrase at a arbitrary point in time.

That being said, I understand the issue you have.

If you only use one of these modules, you could let the Fluencerator know when you want to change, and then let the NEW output from that module trigger the actual change in the other modules.
But that wont work for multiple Fluencerators. And it might not suit your patch to wait for that.

I will look into this. Probably add a second generate input that does it instantly.

PS. I just noticed that the Fluencerator manual, was still under *Coming in v2.1.0* section. I just fixed that.

**Comment by NikolaiVChr** [comment]
Force generate can now be selected in context menu.
